lib/themes/dosomething/paraneue_dosomething/js/images/ImageCropPreview.js

```diff
@@ -0,0 +1,123 @@
+  /**
+   * Creates a front-end preview of a reportback image given crop values.
+   */
+define(function(require) {
+  "use strict";
+
+  var $                      = require("jquery");
+  var $cropPreview           = $("<figure class='reportback__submission-cropped'></figure>");
+  var $reportbackSubmissions = $(".reportback__submissions");
+  var $reportbackUploadBtn   = $(".gigantor");
+
+  /**
+   * Tests if this is the first reportback submission or not.
+   */
+  var firstSubmission = function() {
+    return ($(".reportback__submission-latest").length) ? false : true;
+  };
+
+  /**
+   * This returns the amount to translate an image after it has been rotated so
+   * so that it stays center in it's container.
+   * @TODO - this is a duplicate of a function used in ImageCrop.js. This needs to
+   * be made more DRY. Create a separate "ImageCropUtilities.js" file that can
+   * hold functions that pertain to calculations needed for cropping and rotation.
+   * This should also help clean up the cropping script for easier reading.
+   *
+   * @param {number} width      Width of the image.
+   * @param {number} height     height of the image.
+   * @param {number} degrees    How many degrees the image has been rotated.
+   */
+  var getTranslateDifference = function(width, height, degrees) {
```

Comment:
Maybe eventually pull this function out to an `ImageUtilities` object, or something since it shows up both in **ImageCropPreview.js** and **ImageCrop.js**. Future refactor option :)